diff --git a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/parser/package-info.java b/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/parser/package-info.java
deleted file mode 100644
index a80cf53..0000000
--- a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/parser/package-info.java
+++ /dev/null
@@ -1,4 +0,0 @@
-@Internal
-package me.arcaniax.hdb.libs.kyori.adventure.text.minimessage.internal.parser;
-
-import org.jetbrains.annotations.ApiStatus.Internal;
