diff --git a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/tag/Inserting.java b/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/tag/Inserting.java
deleted file mode 100644
index 46af158..0000000
--- a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/tag/Inserting.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package me.arcaniax.hdb.libs.kyori.adventure.text.minimessage.tag;
-
-import me.arcaniax.hdb.libs.kyori.adventure.text.Component;
-import org.jetbrains.annotations.NotNull;
-
-public interface Inserting extends Tag {
-   @NotNull
-   Component value();
-
-   default boolean allowsChildren() {
-      return true;
-   }
-}
