diff --git a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/tag/ParserDirective.java b/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/tag/ParserDirective.java
deleted file mode 100644
index 4300f71..0000000
--- a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/tag/ParserDirective.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package me.arcaniax.hdb.libs.kyori.adventure.text.minimessage.tag;
-
-import org.jetbrains.annotations.ApiStatus.NonExtendable;
-
-@NonExtendable
-public interface ParserDirective extends Tag {
-   Tag RESET = new ParserDirective() {
-      @Override
-      public String toString() {
-         return "ParserDirective.RESET";
-      }
-   };
-}
