diff --git a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/StorageNBTComponent.java b/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/StorageNBTComponent.java
deleted file mode 100644
index 7c04c80..0000000
--- a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/StorageNBTComponent.java
+++ /dev/null
@@ -1,28 +0,0 @@
-package me.arcaniax.hdb.libs.kyori.adventure.text;
-
-import java.util.stream.Stream;
-import me.arcaniax.hdb.libs.kyori.adventure.key.Key;
-import me.arcaniax.hdb.libs.kyori.examination.ExaminableProperty;
-import org.jetbrains.annotations.Contract;
-import org.jetbrains.annotations.NotNull;
-
-public interface StorageNBTComponent extends NBTComponent<StorageNBTComponent, StorageNBTComponent.Builder>, ScopedComponent<StorageNBTComponent> {
-   @NotNull
-   Key storage();
-
-   @Contract(pure = true)
-   @NotNull
-   StorageNBTComponent storage(@NotNull final Key storage);
-
-   @NotNull
-   @Override
-   default Stream<? extends ExaminableProperty> examinableProperties() {
-      return Stream.concat(Stream.of(ExaminableProperty.of("storage", this.storage())), NBTComponent.super.examinableProperties());
-   }
-
-   interface Builder extends NBTComponentBuilder<StorageNBTComponent, StorageNBTComponent.Builder> {
-      @Contract("_ -> this")
-      @NotNull
-      StorageNBTComponent.Builder storage(@NotNull final Key storage);
-   }
-}
