diff --git a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/event/HoverEventSource.java b/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/event/HoverEventSource.java
deleted file mode 100644
index 92bcee2..0000000
--- a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/event/HoverEventSource.java
+++ /dev/null
@@ -1,20 +0,0 @@
-package me.arcaniax.hdb.libs.kyori.adventure.text.event;
-
-import java.util.function.UnaryOperator;
-import org.jetbrains.annotations.NotNull;
-import org.jetbrains.annotations.Nullable;
-
-public interface HoverEventSource<V> {
-   @Nullable
-   static <V> HoverEvent<V> unbox(@Nullable final HoverEventSource<V> source) {
-      return source != null ? source.asHoverEvent() : null;
-   }
-
-   @NotNull
-   default HoverEvent<V> asHoverEvent() {
-      return this.asHoverEvent(UnaryOperator.identity());
-   }
-
-   @NotNull
-   HoverEvent<V> asHoverEvent(@NotNull final UnaryOperator<V> op);
-}
