diff --git a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/parser/match/MatchedTokenConsumer.java b/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/parser/match/MatchedTokenConsumer.java
deleted file mode 100644
index 3c7fba3..0000000
--- a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/parser/match/MatchedTokenConsumer.java
+++ /dev/null
@@ -1,26 +0,0 @@
-package me.arcaniax.hdb.libs.kyori.adventure.text.minimessage.internal.parser.match;
-
-import me.arcaniax.hdb.libs.kyori.adventure.text.minimessage.internal.parser.TokenType;
-import org.jetbrains.annotations.MustBeInvokedByOverriders;
-import org.jetbrains.annotations.NotNull;
-import org.jetbrains.annotations.UnknownNullability;
-
-public abstract class MatchedTokenConsumer<T> {
-   protected final String input;
-   private int lastIndex = -1;
-
-   public MatchedTokenConsumer(@NotNull final String input) {
-      this.input = inputx;
-   }
-
-   @MustBeInvokedByOverriders
-   public void accept(final int start, final int end, @NotNull final TokenType tokenType) {
-      this.lastIndex = end;
-   }
-
-   public abstract @UnknownNullability T result();
-
-   public final int lastEndIndex() {
-      return this.lastIndex;
-   }
-}
