diff --git a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/parser/match/TokenListProducingMatchedTokenConsumer.java b/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/parser/match/TokenListProducingMatchedTokenConsumer.java
deleted file mode 100644
index 7e01432..0000000
--- a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/parser/match/TokenListProducingMatchedTokenConsumer.java
+++ /dev/null
@@ -1,31 +0,0 @@
-package me.arcaniax.hdb.libs.kyori.adventure.text.minimessage.internal.parser.match;
-
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import me.arcaniax.hdb.libs.kyori.adventure.text.minimessage.internal.parser.Token;
-import me.arcaniax.hdb.libs.kyori.adventure.text.minimessage.internal.parser.TokenType;
-import org.jetbrains.annotations.NotNull;
-
-public final class TokenListProducingMatchedTokenConsumer extends MatchedTokenConsumer<List<Token>> {
-   private List<Token> result = null;
-
-   public TokenListProducingMatchedTokenConsumer(@NotNull final String input) {
-      super(input);
-   }
-
-   @Override
-   public void accept(final int start, final int end, @NotNull final TokenType tokenType) {
-      super.accept(start, end, tokenType);
-      if (this.result == null) {
-         this.result = new ArrayList<>();
-      }
-
-      this.result.add(new Token(start, end, tokenType));
-   }
-
-   @NotNull
-   public List<Token> result() {
-      return this.result == null ? Collections.emptyList() : this.result;
-   }
-}
