diff --git a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/translation/GlobalTranslator.java b/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/translation/GlobalTranslator.java
deleted file mode 100644
index 5a05802..0000000
--- a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/translation/GlobalTranslator.java
+++ /dev/null
@@ -1,39 +0,0 @@
-package me.arcaniax.hdb.libs.kyori.adventure.translation;
-
-import java.util.Locale;
-import me.arcaniax.hdb.libs.kyori.adventure.text.Component;
-import me.arcaniax.hdb.libs.kyori.adventure.text.renderer.TranslatableComponentRenderer;
-import me.arcaniax.hdb.libs.kyori.examination.Examinable;
-import org.jetbrains.annotations.NotNull;
-import org.jetbrains.annotations.ApiStatus.ScheduledForRemoval;
-
-public interface GlobalTranslator extends Translator, Examinable {
-   @NotNull
-   static GlobalTranslator translator() {
-      return GlobalTranslatorImpl.INSTANCE;
-   }
-
-   @Deprecated
-   @ScheduledForRemoval(inVersion = "5.0.0")
-   @NotNull
-   static GlobalTranslator get() {
-      return GlobalTranslatorImpl.INSTANCE;
-   }
-
-   @NotNull
-   static TranslatableComponentRenderer<Locale> renderer() {
-      return GlobalTranslatorImpl.INSTANCE.renderer;
-   }
-
-   @NotNull
-   static Component render(@NotNull final Component component, @NotNull final Locale locale) {
-      return renderer().render(component, locale);
-   }
-
-   @NotNull
-   Iterable<? extends Translator> sources();
-
-   boolean addSource(@NotNull final Translator source);
-
-   boolean removeSource(@NotNull final Translator source);
-}
