diff --git a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/parser/node/TextNode.java b/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/parser/node/TextNode.java
deleted file mode 100644
index da0d4dd..0000000
--- a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/parser/node/TextNode.java
+++ /dev/null
@@ -1,21 +0,0 @@
-package me.arcaniax.hdb.libs.kyori.adventure.text.minimessage.internal.parser.node;
-
-import me.arcaniax.hdb.libs.kyori.adventure.text.minimessage.internal.parser.Token;
-import me.arcaniax.hdb.libs.kyori.adventure.text.minimessage.internal.parser.TokenParser;
-import org.jetbrains.annotations.NotNull;
-import org.jetbrains.annotations.Nullable;
-
-public final class TextNode extends ValueNode {
-   private static boolean isEscape(final int escape) {
-      return escape == 60 || escape == 92;
-   }
-
-   public TextNode(@Nullable final ElementNode parent, @NotNull final Token token, @NotNull final String sourceMessage) {
-      super(parent, token, sourceMessage, TokenParser.unescape(sourceMessage, token.startIndex(), token.endIndex(), TextNode::isEscape));
-   }
-
-   @Override
-   String valueName() {
-      return "TextNode";
-   }
-}
