diff --git a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/parser/node/ValueNode.java b/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/parser/node/ValueNode.java
deleted file mode 100644
index dad2747..0000000
--- a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/parser/node/ValueNode.java
+++ /dev/null
@@ -1,36 +0,0 @@
-package me.arcaniax.hdb.libs.kyori.adventure.text.minimessage.internal.parser.node;
-
-import java.util.Objects;
-import me.arcaniax.hdb.libs.kyori.adventure.text.minimessage.internal.parser.Token;
-import org.jetbrains.annotations.NotNull;
-import org.jetbrains.annotations.Nullable;
-
-public abstract class ValueNode extends ElementNode {
-   private final String value;
-
-   ValueNode(@Nullable final ElementNode parent, @Nullable final Token token, @NotNull final String sourceMessage, @NotNull final String value) {
-      super(parent, token, sourceMessage);
-      this.value = valuex;
-   }
-
-   abstract String valueName();
-
-   @NotNull
-   public String value() {
-      return this.value;
-   }
-
-   @NotNull
-   @Override
-   public Token token() {
-      return Objects.requireNonNull(super.token(), "token is not set");
-   }
-
-   @NotNull
-   @Override
-   public StringBuilder buildToString(@NotNull final StringBuilder sb, final int indent) {
-      char[] var3 = this.ident(indent);
-      sb.append(var3).append(this.valueName()).append("('").append(this.value).append("')\n");
-      return sb;
-   }
-}
