diff --git a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/serializer/ClaimConsumer.java b/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/serializer/ClaimConsumer.java
deleted file mode 100644
index 9e3f186..0000000
--- a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/serializer/ClaimConsumer.java
+++ /dev/null
@@ -1,13 +0,0 @@
-package me.arcaniax.hdb.libs.kyori.adventure.text.minimessage.internal.serializer;
-
-import org.jetbrains.annotations.NotNull;
-
-public interface ClaimConsumer {
-   void style(@NotNull final String claimKey, @NotNull final Emitable styleClaim);
-
-   boolean component(@NotNull final Emitable componentClaim);
-
-   boolean styleClaimed(@NotNull final String claimId);
-
-   boolean componentClaimed();
-}
