diff --git a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/serializer/StyleClaim.java b/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/serializer/StyleClaim.java
deleted file mode 100644
index 07e0beb..0000000
--- a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/serializer/StyleClaim.java
+++ /dev/null
@@ -1,39 +0,0 @@
-package me.arcaniax.hdb.libs.kyori.adventure.text.minimessage.internal.serializer;
-
-import java.util.Objects;
-import java.util.function.BiConsumer;
-import java.util.function.Function;
-import java.util.function.Predicate;
-import me.arcaniax.hdb.libs.kyori.adventure.text.format.Style;
-import org.jetbrains.annotations.NotNull;
-import org.jetbrains.annotations.Nullable;
-
-public interface StyleClaim<V> {
-   @NotNull
-   static <T> StyleClaim<T> claim(
-      @NotNull final String claimKey, @NotNull final Function<Style, @Nullable T> lens, @NotNull final BiConsumer<T, TokenEmitter> emitable
-   ) {
-      return claim(claimKey, lens, $ -> true, emitable);
-   }
-
-   @NotNull
-   static <T> StyleClaim<T> claim(
-      @NotNull final String claimKey,
-      @NotNull final Function<Style, @Nullable T> lens,
-      @NotNull final Predicate<T> filter,
-      @NotNull final BiConsumer<T, TokenEmitter> emitable
-   ) {
-      return new StyleClaimImpl(
-         Objects.requireNonNull(claimKey, "claimKey"),
-         Objects.requireNonNull((Function<Style, V>)lens, "lens"),
-         Objects.requireNonNull((Predicate<V>)filter, "filter"),
-         Objects.requireNonNull((BiConsumer<V, TokenEmitter>)emitable, "emitable")
-      );
-   }
-
-   @NotNull
-   String claimKey();
-
-   @Nullable
-   Emitable apply(@NotNull final Style style);
-}
