diff --git a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/serializer/TokenEmitter.java b/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/serializer/TokenEmitter.java
deleted file mode 100644
index 62af65a..0000000
--- a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/text/minimessage/internal/serializer/TokenEmitter.java
+++ /dev/null
@@ -1,36 +0,0 @@
-package me.arcaniax.hdb.libs.kyori.adventure.text.minimessage.internal.serializer;
-
-import me.arcaniax.hdb.libs.kyori.adventure.text.Component;
-import org.jetbrains.annotations.NotNull;
-
-public interface TokenEmitter {
-   @NotNull
-   TokenEmitter tag(@NotNull final String token);
-
-   @NotNull
-   TokenEmitter selfClosingTag(@NotNull final String token);
-
-   @NotNull
-   default TokenEmitter arguments(@NotNull final String... args) {
-      for (String arg : args) {
-         this.argument(arg);
-      }
-
-      return this;
-   }
-
-   @NotNull
-   TokenEmitter argument(@NotNull final String arg);
-
-   @NotNull
-   TokenEmitter argument(@NotNull final String arg, @NotNull final QuotingOverride quotingPreference);
-
-   @NotNull
-   TokenEmitter argument(@NotNull final Component arg);
-
-   @NotNull
-   TokenEmitter text(@NotNull final String text);
-
-   @NotNull
-   TokenEmitter pop();
-}
