diff --git a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/key/InvalidKeyException.java b/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/key/InvalidKeyException.java
deleted file mode 100644
index 4be9ddf..0000000
--- a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/key/InvalidKeyException.java
+++ /dev/null
@@ -1,26 +0,0 @@
-package me.arcaniax.hdb.libs.kyori.adventure.key;
-
-import org.jetbrains.annotations.NotNull;
-import org.jetbrains.annotations.Nullable;
-
-public final class InvalidKeyException extends RuntimeException {
-   private static final long serialVersionUID = -5413304087321449434L;
-   private final String keyNamespace;
-   private final String keyValue;
-
-   InvalidKeyException(@NotNull final String keyNamespace, @NotNull final String keyValue, @Nullable final String message) {
-      super(message);
-      this.keyNamespace = keyNamespacex;
-      this.keyValue = keyValuex;
-   }
-
-   @NotNull
-   public final String keyNamespace() {
-      return this.keyNamespace;
-   }
-
-   @NotNull
-   public final String keyValue() {
-      return this.keyValue;
-   }
-}
