diff --git a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/key/KeyPattern.java b/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/key/KeyPattern.java
deleted file mode 100644
index b20f297..0000000
--- a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/key/KeyPattern.java
+++ /dev/null
@@ -1,28 +0,0 @@
-package me.arcaniax.hdb.libs.kyori.adventure.key;
-
-import java.lang.annotation.Documented;
-import java.lang.annotation.ElementType;
-import java.lang.annotation.Retention;
-import java.lang.annotation.RetentionPolicy;
-import java.lang.annotation.Target;
-import org.intellij.lang.annotations.Pattern;
-
-@Documented
-@Retention(RetentionPolicy.CLASS)
-@Target({ElementType.FIELD, ElementType.LOCAL_VARIABLE, ElementType.METHOD, ElementType.PARAMETER})
-@Pattern("(?:([a-z0-9_\\-.]+:)?|:)[a-z0-9_\\-./]+")
-public @interface KeyPattern {
-   @Documented
-   @Retention(RetentionPolicy.CLASS)
-   @Target({ElementType.FIELD, ElementType.LOCAL_VARIABLE, ElementType.METHOD, ElementType.PARAMETER})
-   @Pattern("[a-z0-9_\\-.]+")
-   @interface Namespace {
-   }
-
-   @Documented
-   @Retention(RetentionPolicy.CLASS)
-   @Target({ElementType.FIELD, ElementType.LOCAL_VARIABLE, ElementType.METHOD, ElementType.PARAMETER})
-   @Pattern("[a-z0-9_\\-./]+")
-   @interface Value {
-   }
-}
