diff --git a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/bossbar/BossBarViewer.java b/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/bossbar/BossBarViewer.java
deleted file mode 100644
index 2b7bc16..0000000
--- a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/bossbar/BossBarViewer.java
+++ /dev/null
@@ -1,9 +0,0 @@
-package me.arcaniax.hdb.libs.kyori.adventure.bossbar;
-
-import org.jetbrains.annotations.NotNull;
-import org.jetbrains.annotations.UnmodifiableView;
-
-public interface BossBarViewer {
-   @NotNull
-   @UnmodifiableView Iterable<? extends BossBar> activeBossBars();
-}
