diff --git a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/identity/NilIdentity.java b/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/identity/NilIdentity.java
deleted file mode 100644
index 0c466e5..0000000
--- a/src/main/java/me/arcaniax/hdb/libs/kyori/adventure/identity/NilIdentity.java
+++ /dev/null
@@ -1,31 +0,0 @@
-package me.arcaniax.hdb.libs.kyori.adventure.identity;
-
-import java.util.UUID;
-import org.jetbrains.annotations.NotNull;
-import org.jetbrains.annotations.Nullable;
-
-final class NilIdentity implements Identity {
-   static final UUID NIL_UUID = new UUID(0L, 0L);
-   static final Identity INSTANCE = new NilIdentity();
-
-   @NotNull
-   @Override
-   public UUID uuid() {
-      return NIL_UUID;
-   }
-
-   @Override
-   public String toString() {
-      return "Identity.nil()";
-   }
-
-   @Override
-   public boolean equals(@Nullable final Object that) {
-      return this == that;
-   }
-
-   @Override
-   public int hashCode() {
-      return 0;
-   }
-}
